ultraLatency.py

```diff
@@ -188,10 +187,28 @@
     sys.stdout.flush()
     startScanModule(ohboard, options.gtx, useUltra=True, debug=options.debug)
     printScanConfiguration(ohboard, options.gtx, useUltra=True, debug=options.debug)
+    amc13nL1A = (amc13board.read(amc13board.Board.T1, "STATUS.GENERAL.L1A_COUNT_HI") << 32) | (amc13board.read(amc13board.Board.T1, "STATUS.GENERAL.L1A_COUNT_LO"))
+    amcnL1A = getAMCL1Acount(amcboard)
+    ohnL1A = getLinkL1Acount(amcboard,options.gtx)
+    print "Initial L1A counts:"
+    print "AMC13: %s"%(amc13nL1A)
+    print "AMC: %s"%(amcnL1A)
+    print "OH%s: %s"%(link,ohnL1A)
     sys.stdout.flush()
-    scanData = getUltraScanResults(ohboard, options.gtx, LATENCY_MAX - LATENCY_MIN + 1, options.debug)
+    scanData = getUltraScanResults(ohboard, options.gtx, LATENCY_MAX - LATENCY_MIN + 1, options.debug, int(options.nevts), True)
 
     print("Done scanning, processing output")
+    amc13nL1Af = (amc13board.read(amc13board.Board.T1, "STATUS.GENERAL.L1A_COUNT_HI") << 32) | (amc13board.read(amc13board.Board.T1, "STATUS.GENERAL.L1A_COUNT_LO"))
+    amcnL1Af = getAMCL1Acount(amcboard)
+    ohnL1Af = getLinkL1Acount(amcboard,options.gtx)
+    print "Final L1A counts:"
+    print "AMC13: %s, difference %s"%(amc13nL1Af,amc13nL1Af-amc13nL1A)
+    print "AMC: %s, difference %s"%(amcnL1Af,amcnL1Af-amcnL1A)
+    print "OH%s: %s, difference %s"%(link,ohnL1Af,ohnL1Af-ohnL1A)
+
+    for i in range(24):
+      print "Number of CRC errors for VFAT%s on link %s is %s"%(options.gtx, i, readRegister("GEM_AMC.OH.OH%d.COUNTERS.CRC.INCORRECT.VFAT%d"%(options.gtx,i)))
```

Comment:
you mean this?
`eagle34 > read GEM_AMC.OH.OH0.COUNTERS.CRC.INCORRECT.VFAT0
0x650280f0 r    GEM_AMC.OH.OH0.COUNTERS.CRC.INCORRECT.VFAT0             0x003b2c95
eagle34 > write GEM_AMC.OH.OH0.COUNTERS.CRC.INCORRECT.VFAT0.Reset 1
0x00000001(1)   written to GEM_AMC.OH.OH0.COUNTERS.CRC.INCORRECT.VFAT0.Reset
eagle34 > read GEM_AMC.OH.OH0.COUNTERS.CRC.INCORRECT.VFAT0
0x650280f0 r    GEM_AMC.OH.OH0.COUNTERS.CRC.INCORRECT.VFAT0             0x00000000`